docs/developer/events.rst

```diff
@@ -15,6 +15,40 @@ showInfoPanel Object Show the info panel in the right.
 hideInfoPanel NULL   Hide the info panel displayed in the right. 
 ============= ====== =========================================== 
 
+The `k-info-panel` can be triggered by a `k-button` or a `k-action-menu` or
+anything that calls the method created to set-up a new `k-info-panel`.
+
+**Example**
+
+.. code-block:: html
+
+    <template>
+        ...
+    </template>
+
+    <script>
+    /* All the javascript methods are optional */
+    module.exports = {
+        methods: {
+            showInfoPanel()  {
+                var content = {
+                    "component": 'search-hosts',
+                    "content": {}, /* Content used in the component */

```

Comment:
Can we include an example of `content` here?

Reply:
yes, sure